pages/api/spotify.js

```diff
@@ -1,22 +1,29 @@
 require('dotenv').config()
-const Spotify = require('spotify-web-api-node')
-const request = require('request-promise')
+import request from 'request-promise'
+import { parse } from 'query-string'
 
+const Spotify = require('spotify-web-api-node')
 const spotify = new Spotify()
 spotify.setRefreshToken(process.env.REFRESH_TOKEN)
 spotify.setClientId(process.env.CLIENT_ID)
 spotify.setClientSecret(process.env.CLIENT_SECRET)
 
-const refreshToken = () =>
-  new Promise((res, rej) => {
+const refreshToken = async () => {
+  return new Promise((res, rej) => {
```

Comment:
This is a longer version of the same syntax, the return word & braces can be removed